packages/core/test/integration/query-interface.test.js

```diff
@@ -21,15 +20,10 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
   if (dialect.supports.schemas) {
     describe('dropAllSchema', () => {
       it('should drop all schema', async function () {
-        await this.queryInterface.dropAllSchemas({
-          skip: [this.sequelize.config.database],
-        });
+        await Support.dropTestSchemas(this.sequelize);
```

Comment:
this is the only change I'm opposed to as `dropAllSchema` is what is being tested and we can't guarantee we won't change `dropTestSchemas` in the future

Reply:
Woops, you're right. Not sure how I looked over that 😅

Reply:
Fixed